src/main/java/org/folio/circulation/resources/LoanCollectionResource.java

```diff
@@ -345,4 +347,21 @@ private ItemNotFoundValidator createItemNotFoundValidator(Loan loan) {
         String.format("No item with ID %s could be found", loan.getItemId()),
         ITEM_ID, loan.getItemId()));
   }
+
+  private Result<LoanAndRelatedRecords> refuseWhenItemHasDeclaredLostStatus(
```

Comment:
I think this can be shortened by removing status:

```suggestion
  private Result<LoanAndRelatedRecords> refuseWhenItemIsDeclaredLost(
```

Reply:
Changed